src/Nancy.Serialization.JsonNet/Helpers.cs

```diff
@@ -23,10 +23,10 @@ public static bool IsJsonType(string contentType)
 
             var contentMimeType = contentType.Split(';')[0];
 
-            return contentMimeType.Equals("application/json", StringComparison.InvariantCultureIgnoreCase) ||
-                   contentMimeType.Equals("text/json", StringComparison.InvariantCultureIgnoreCase) ||
-                  (contentMimeType.StartsWith("application/vnd", StringComparison.InvariantCultureIgnoreCase) &&
-                   contentMimeType.EndsWith("+json", StringComparison.InvariantCultureIgnoreCase));
+            return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
+                   contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
+                  (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&

```

Comment:
While you're at it, could you remove the `contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase)` check? I don't think it's necessary.